client/settings/express-checkout/index.js

```diff
@@ -210,52 +211,77 @@ const ExpressCheckout = () => {
 						<li className="express-checkout has-icon-border">
 							<div className="express-checkout__checkbox loadable-checkbox label-hidden">
 								<CheckboxControl
-									label={ 'Stripe Link' }
+									label={ 'Link by Stripe' }
```

Comment:
A super minor nitpick, if I may - `Stripe Link` earlier, as a brand would read same across all languages and thats why perhaps we did not have a translation escape tag added on it.  Whereas for the term `Link by Stripe` , we may have scope for translation because of the word `by` in it.  

Eg: On a Spanish site, `Link by Stripe` may look slightly off/ something left out of translation.

Reply:
updated in [eeeacc1](https://github.com/Automattic/woocommerce-payments/pull/4852/commits/eeeacc10cf128c708c92cf5ca1d8d37c557bc5f0) . thank you!!